include/oneapi/dnnl/dnnl.h

```diff
@@ -271,24 +271,53 @@ dnnl_status_t DNNL_API dnnl_primitive_attr_clone(
 ///     otherwise.
 dnnl_status_t DNNL_API dnnl_primitive_attr_destroy(dnnl_primitive_attr_t attr);
 
-/// Returns probability for output dropout primitive attribute.
+/// Sets dropout primitive attribute.
 ///
 /// @param attr Primitive attributes.
-/// @param dropout_desc Output dropout memory descriptor
+/// @param mask_desc Output memory descriptor for dropout masks
 /// @returns #dnnl_success on success and a status describing the error
 ///     otherwise.
 dnnl_status_t DNNL_API dnnl_primitive_attr_get_dropout(
-        const_dnnl_primitive_attr_t attr,
-        const_dnnl_memory_desc_t *dropout_desc);
+        const_dnnl_primitive_attr_t attr, const_dnnl_memory_desc_t *mask_desk);
 
-/// Sets probability for output dropout primitive attribute.
+/// Sets dropout primitive attribute.
 ///
 /// @param attr Primitive attributes.
-/// @param dropout_desc Output dropout memory descriptor
+/// @param mask_desc Output memory descriptor for dropout masks
 /// @returns #dnnl_success on success and a status describing the error
 ///     otherwise.
 dnnl_status_t DNNL_API dnnl_primitive_attr_set_dropout(
-        dnnl_primitive_attr_t attr, const_dnnl_memory_desc_t dropout_desc);
+        dnnl_primitive_attr_t attr, const_dnnl_memory_desc_t mask_desc);
+
+/// Gets dropout primitive attribute parameters.
+///
+/// @param attr Primitive attributes.
+/// @param mask_desc Output memory descriptor for dropout masks
+/// @param seed_dt Datatype used for seed argument.
+/// @param use_offset If true, an offset argument is passed at
+///        execution and used in random number generation
+/// @param use_host_scalars If true, probability, seed and offset are
+////       passed as host_scalar memory objects.
```

Comment:
It seems getter arguments should be documented differently as they don't affect the library behavior.